portfolio/src/main/webapp/index.html

```diff
@@ -80,10 +80,9 @@ <h2>Comments</h2>
       <form action="/data" method="post" target="_self">
         <textarea name="comment" placeholder="Comment..."></textarea>
         <br/><br/>
-        <input type="submit" value="Submit">
+        <input type="submit" value="Submit" >
```

Comment:
Nit: It's a good practice to avoid making non-functional (e.g. formatting) changes. Many projects apply an auto-formatter to their code, not only because the result is more consistent, but also because code review is easier if all diffs are semantically meaningful.